src/bgfx.cpp

```diff
@@ -317,6 +317,7 @@ namespace bgfx
 	std::atomic<bool> g_suspendRequested = false;
 	std::atomic<bool> g_suspendComplete = false;
 	std::atomic<bool> g_resumeRequested = false;
+	std::atomic<bool> g_resumeComplete = true;
```

Comment:
bgfx is not using STL.

Reply:
I'm removing the pull request. Will resubmit once I have a chance to replace std::atomic and the game passes certicication.